app/controllers/events_controller.rb

```diff
@@ -4,6 +4,15 @@ class EventsController < InheritedResources::Base
   skip_load_and_authorize_resource only: [:index, :show]
   before_filter :set_types
 
+  def participants
+    @breadcrumbs = [
+      { label: 'events', url: events_path },
+      { label: @event.name, url: event_path(@event) },
+      { label: 'participants' }
+    ]
+    @users = @event.users.travel
```

Comment:
You're still not using accessible_by(current_ability), this can cause some privacy issues if you access data without checking the ability.